src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -15,19 +16,21 @@ public static long countWords() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        long count = 0;
-        for (String w : words) {
-            if (w.length() > 12) count++;
-        }
-        return count;
+        return words.stream().filter(w -> w.length() > 12).count();
```

Comment:
```suggestion
        return words.stream()
                .filter(word -> word.length() > 12)
                .count();
```
이렇게 줄을 정리하면 더 읽기가 좋아질 것 같아요. :)